Automattic/wp-calypso
# Devdocs: Use Webpack Loader to load READMEs

Preparation for #35809.
Merge #36167 before this PR.

I think this is in the spirit of the previous architecture, but leverages modern technologies better: We continue to load and render those individual READMEs, and can even serve them over the network, thanks to Webpack's lazy loading feature. 

#### Changes proposed in this Pull Request

* Install markdown-loader, and use it to load .md files

#### Testing instructions

- Go to `http://calypso.localhost:3000/devdocs/design/button`, and verify that the README is rendered correctly at the bottom of the page (compare to https://wpcalypso.wordpress.com/devdocs/design/button).
- Verify that other devdocs links in the sidebar, such as `http://calypso.localhost:3000/devdocs/docs/guide/index.md`, also still work.

#### TODO

Fix `http://calypso.localhost:3000/devdocs/.github/CONTRIBUTING.md`. Probably broken because of the hidden `.github` directory :slightly_frowning_face: 
**Update:** Looks like we might be able to just move it to `docs/` and it'll still be used by GitHub! :tada: 

> To help your project contributors do good work, you can add a file with contribution guidelines to your project repository's root, `docs`, or `.github` folder. When someone opens a pull request or creates an issue, they will see a link to that file.

[(Source)](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors)

I guess I'll file a separate PR. **Update:** #36167
